HPCC-13431 DOCS:New functionality in install-cluster.sh

g-pan opened the pull request:
FIX HPCC-13431 DOCS:New functionality in install-cluster.sh
Document the New functionality of install-cluster.sh
(in installing and running the HPCC platform book)

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com

@JamesDeFabia please review

g-pan committed:
HPCC-13431 DOCS:New functionality in install-cluster.sh

FIX HPCC-13431 DOCS:New functionality in install-cluster.sh
Document the New functionality of install-cluster.sh
(in installing and running the HPCC platform book)

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
Since you can choose only  -k or -p , it should be noted as:
 [-k | -p] in syntax diagram. 

And, it wouldn't hurt to mention that you can only choose one or the other in the description.

g-pan committed:
HPCC-13431 DOCS:New functionality in install-cluster.sh

FIX HPCC-13431 DOCS:New functionality in install-cluster.sh
Update the New functionality of install-cluster.sh
(in installing and running the HPCC platform book)

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-13431 DOCS:New functionality in install-cluster.sh

FIX HPCC-13431 DOCS:New functionality in install-cluster.sh
Update the New functionality of install-cluster.sh
(in installing and running the HPCC platform book)

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>
